internal/collector/instance.go

```diff
@@ -41,11 +41,13 @@ func AddToPod(
 	outPod *corev1.PodSpec,
 	volumeMounts []corev1.VolumeMount,
 	sqlQueryPassword string,
+	includeLogrotate bool,
```

Comment:
❓ Do you expect we'll stop passing in `false` here eventually?

Reply:
I think we will still pass `false` for pgAdmin because it has its own log rotation mechanism.

Reply:
Ah! right.